operations/helm/charts/mimir-distributed/templates/alertmanager/alertmanager-statefulset.yaml

```diff
@@ -18,6 +18,10 @@ spec:
     {{- toYaml .Values.alertmanager.statefulStrategy | nindent 4 }}
   serviceName: {{ template "mimir.fullname" . }}-alertmanager
   {{- if .Values.alertmanager.persistentVolume.enabled }}
+  {{- if .Values.alertmanager.persistentVolume.volumeClaimTemplates }}
```

Comment:
To be more consistent (with extraEnv, extraEnvFrom, etc, etc) the new value (if we decide to do add it) should be called extraVolumeClaimTemplates and it should be added after the existing conditional section. 
If could be added from line 42 (original) as something like:
```
    {{- with .Values.alertmanager.persistentVolume.extraVolumeClaimTemplates }}
    {{- toYaml . | nindent 4 }}
    {{- end }}
```
also for the other components.